packages/expo-polyfills/package.json

```diff
@@ -49,21 +49,21 @@
   "devDependencies": {
     "@medplum/core": "3.1.5",
     "@types/base-64": "1.0.2",
-    "@types/react": "18.3.1",
+    "@types/react": "18.3.2",
     "@types/text-encoding": "0.0.39",
-    "esbuild": "0.20.2",
-    "esbuild-node-externals": "1.13.0",
+    "esbuild": "0.21.2",
+    "esbuild-node-externals": "1.13.1",
     "jest": "29.7.0",
     "jest-expo": "50.0.4",
-    "rimraf": "5.0.5",
+    "rimraf": "5.0.7",
     "ts-jest": "29.1.2"
   },
   "peerDependencies": {
     "@medplum/core": "3.1.5",
     "expo": "*",
-    "expo-crypto": "^12.6.0",
-    "expo-secure-store": "^12.3.1",
-    "expo-standard-web-crypto": "^1.7.0",
+    "expo-crypto": "^13.0.0",
```

Comment:
Not convinced our test coverage is enough to prove these are working as intended, we'll probably want to manually test on real mobile devices before merging this